R/stage_nwis_ts.R

```diff
@@ -113,13 +117,18 @@ stage_nwis_ts <- function(sites, var, times, folder = tempdir(), verbose = FALSE
     for (i in 1:length(un_sites)){
       
       site <- parse_site_name(un_sites[i])
-      site_data <- filter(nwis_data, site_no == site) %>%
-        select_('DateTime', var) %>%
+      site_data <- filter(nwis_data, site_no == site)
+      which.var <- select(site_data, -DateTime, -site_no) %>% 
```

Comment:
if there is only one descriptor name, the most common case, then .$key[1] is NA, as opposed to .$key is the descriptor (column) name